test/carbon/Strategies.ts

```diff
@@ -86,7 +86,7 @@ describe('Strategy', () => {
         // prepare variables
         const _params = { ...params };
         const order = _params.order ? _params.order : generateTestOrder();
-        const secondOrder = order ? order : generateTestOrder();
+        const secondOrder = order || generateTestOrder();
```

Comment:
Nitpick: I usually prefer `??` (over `||`) because it won't override falsy values (e.g., `0` or `""`), but only undefined/optional values

Reply:
fixed